packages/react-native-worklets/android/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+project(Worklets)
```

Comment:
`Worklets` of `worklets`?

Reply:
What do you mean? `CMakeLists.txt` in reanimated directory specifies `Reanimated` project.